iep-nflxenv/src/main/resources/reference.conf

```diff
@@ -18,6 +18,7 @@ netflix.iep.env {
   region = "us-east-1"
   region = ${?EC2_REGION}
   region = ${?NETFLIX_REGION}
+  region = ${?NETFLIX_HOME_REGION}
```

Comment:
I think this setting should probably remain the actual region something is running in rather than the infrastructure region. For example, it is also used for determining the region to use for AWS services.

Reply:
Hm, good point. It would probably be best to keep this specific override in the Java thin client configuration - I can't think of many more places where we would use such a value from configuration.